src/status_im/chat/events.cljs

```diff
@@ -194,10 +198,10 @@
                         {}
                         all-stored-chats)]
       (handlers-macro/merge-fx cofx
-                         {:db (assoc db :chats chats)}
-                         (init-console-chat)
-                         (group.events/add-default-groups)
-                         (add-default-contacts)))))
+                               {:db (assoc db :chats chats)}
```

Comment:
@janherich let's just skip re-indentation, we can fix this in a separate PR. By fixing `merge-fx` you don't fix all indentations related problems in project but this adds a lot of noise (PR is big even without this).

Reply:
Optionally we can put re-indentation into separate commit.

Reply:
I'm little bit uneasy about that, as the indentation is now correct and was wrong previously (when `handlers` -> `handlers-macro` change was made) + github highlights indentation only changes anyway, but if it's necessary, we can do that in such two step approach, sounds like a lot of unnecessary work IMO.

Reply:
> sounds like a lot of unnecessary work IMO
yep i agree, i wouldn't touch them in the first place because of this. You can't separate visually changes which are only indentations and which change functionality, this means that you literally need to read all `+730 −1,009` in order to review, idk, 100-200 loc. That's a lot of unnecessary work IMO.